swagger.yml

```
@@ -1163,7 +1163,7 @@ paths:
       tags:
         - "Tenant API - Content"
       summary: Send content to a recipient (user or company)
-      operationId: Send content
+      operationId: post_v1_tenant_key_content
```

I'll change it back. We can update all of them later instead if we'd like.

```suggestion
      operationId: Send content
```